backend/src/v5/routes/teamspaces/teamspaces.js

```diff
@@ -174,6 +200,85 @@ const establishRoutes = () => {
 	*/
 	router.get('/:teamspace/avatar', hasAccessToTeamspace, getAvatar);
 
+	/**
+	* @openapi
+	* /teamspaces/{teamspace}/quota:
+	*   get:
+	*     description: Gets quota information about a user
+	*     tags: [Teamspaces]
+	*     parameters:
+   	*       - name: teamspace
+	*         description: name of teamspace
+	*         in: path
+	*         required: true
+	*         schema:
+	*           type: string
+	*     operationId: getQuotaInfo
+	*     responses:
+	*       401:
+	*         $ref: "#/components/responses/notLoggedIn"
+	*       200:
+	*         description: Gets the quota information of the user
+	*         content:
+	*           application/json:
+	*             schema:
+	*               type: object
+	*               properties:
```

Comment:
This request came in the design meeting the other day, can we add 2 more things to this:

1. expiry date of the quota
2. whether the user is a paid user (i.e. only have the basic quota) 


So end result:
````js
{
   freeTier: false, //or true for teamspaces with only base sub
   expiryDate: 1233445 // timestamp in epoch, missing/undefined is fine for base and never expiring accounts
   data: {...},
   seats: {...}
}
````